osu.Game/Rulesets/Mods/ModAlternate.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Bindings;
+using osu.Framework.Input.Events;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.UI;
+using osu.Game.Screens.Play;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModAlternate : Mod
+    {
+        public override string Name => @"Alternate";
+        public override string Acronym => @"AL";
+        public override string Description => @"Never hit the same key twice!";
```

Comment:
not sure how i missed that in the original PR, but this description is just... not right, as it can be understood as never being allowed to press a key more than once. i suggest the following clarification:

```suggestion
        public override string Description => @"Don't use the same key twice in a row!";
```

Reply:
Might only be me, but `Never` reads better than `Don't` in this case. I think going with `Never hit the same key twice in a row!` works best.